scripts/export-realesrgan-onnx.py

```
@@ -0,0 +1,82 @@
+# (filename-stem, release tag) — pinned release assets (immutable).
+MODELS = {
+    "x4": ("RealESRGAN_x4plus", "v0.1.0"),
+    "x2": ("RealESRGAN_x2plus", "v0.2.1"),
+}
+BASE_URL = "https://github.com/xinntao/Real-ESRGAN/releases/download/{tag}/{name}.pth"
+
+
+def download(name: str, tag: str, dest: str) -> None:
+    url = BASE_URL.format(tag=tag, name=name)
+    print(f"  downloading {url}")
+    urllib.request.urlretrieve(url, dest)
```

Ruff (0.15.17)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)